helm/libcoder/templates/_rbac.yaml

```diff
@@ -1,64 +1,57 @@
-{{- define "libcoder.rbac.tpl" -}}
-{{- if .Values.coder.serviceAccount.workspacePerms }}
+{{- define "libcoder.rbac.forNamespace" -}}
+{{- $top := .Top -}}
+{{- $ns := .NS -}}
+{{- $nsPerms := ternary .workspacePerms $top.Values.coder.serviceAccount.workspacePerms (hasKey . "workspacePerms") -}}
+{{- $nsDeploy := ternary .enableDeployments $top.Values.coder.serviceAccount.enableDeployments (hasKey . "enableDeployments") -}}
+{{- $nsExtra  := ternary .extraRules $top.Values.coder.serviceAccount.extraRules (hasKey . "extraRules") -}}
+{{- if $nsPerms }}
```

Comment:
Maybe this should be `if $nsPerms || $nsDeploy || $nsExtra` or equivalent, and wrap the `libcoder.rbac.rules.basic` in an `if $nsPerms`

Reply:
Would allow for only deployment perms or only extra rules